src/lib.rs

```diff
@@ -217,9 +217,11 @@ pub use self::encoder::TextEncoder;
 pub use self::encoder::{PROTOBUF_FORMAT, TEXT_FORMAT};
 pub use self::errors::{Error, Result};
 pub use self::gauge::{Gauge, GaugeVec, IntGauge, IntGaugeVec};
+#[deprecated(since = "0.12.0", note = "Please use DEFAULT_FLOAT_BUCKETS")]
 pub use self::histogram::DEFAULT_BUCKETS;
```

Comment:
Considering that Histogram is called Histogram instead of FloatHistogram, I suggest to keep `DEFAULT_BUCKETS` and not being `DEFAULT_FLOAT_BUCKETS`. What do you think?